cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,29 @@
+/// <reference types="cypress" />
+
+const { generateUser } = require('../support/generate');
```

Comment:
```suggestion
const { generateUser } = require('../support/generate');
```
this file is absent in your pr, add it